GoogleSheetI18n/Core/I18nGoogleClient.cs

```diff
@@ -29,6 +33,33 @@ public async Task<IList<I18nSheet>> GetSheets(string spreadsheetId)
                 .ToArray();
         }
 
+        public string GetExtractedTranslations(string pageName, List<object> possibleKeys)
```

Comment:
Could you move the method to separate class `CodeTranslations`. And move validation logic to separate class.
In other words, it is better to separate retrieving data and validation logics.